cpp/src/cugraph.cu

```diff
@@ -70,12 +70,16 @@ void cpy_column_view(const gdf_column *in, gdf_column *out) {
 
 gdf_error gdf_adj_list_view(gdf_graph *graph, const gdf_column *offsets,
                                  const gdf_column *indices, const gdf_column *edge_data) {
+  //This function returns an error if this graph object has at least one graph
+  //representation to prevent a single object storing two different graphs.
+  GDF_REQUIRE( ((graph->edgeList == nullptr) && (graph->adjList == nullptr) &&
+    (graph->transposedAdjList == nullptr)), GDF_INVALID_API_CALL);
+  GDF_REQUIRE( (graph->transposedAdjList == nullptr) , GDF_INVALID_API_CALL);
```

Comment:
redundant

Reply:
Thanks for finding this and I made a fix!